napi.h

```diff
@@ -850,18 +846,16 @@ namespace Napi {
     void Queue();
     void Cancel();
 
-    virtual void Execute() = 0;
-    virtual void WorkComplete();
-
     ObjectReference& Persistent();
 
   protected:
     explicit AsyncWorker(const Function& callback);
```

Comment:
Were you still planning on having an overload that allowed the user to pass in the this pointer for use with the callbacks?